lib/tasks/add-missing-files.js

```diff
@@ -17,7 +17,7 @@ const files = [
 
 const jsFiles = [['babel.config.cjs', 'files/js/babel.config.cjs']];
 
-const tsFiles = [['babel.config.cjs', 'files/ts/babel.config.cjs']];
+const tsFiles = [['babel.config.cjs', 'files/ts/babel.config.cjs', 'files-override/ts/types/index.d.ts']];
```

Comment:
Create `files-override/ts/types/index.d.ts` directly from the Vite app blueprint if it doesn't exist.